### DISCUSSION TITLE
是否会适配Android15呢

### DISCUSSION THREAD

**Discussion by Iimeko** [xfqwdsj/IAmNotADeveloper#81] [Q&A] [answered]
*Please ask a question using [Discussions](https://github.com/xfqwdsj/IAmNotADeveloper/discussions)*
更新Android15之后公司开发的打卡app会检测到开发者模式，不知道会不会做适配

**Comment by Iimeko** [comment]
![Uploading Screenshot_20241023-092239.png…]()

**Comment by xfqwdsj** [comment, marked as answer]
首先，提问请使用 Discussions。

目前，本模块从设计上不依赖特定 Android 版本，所以不存在“适配”Android 版本的问题。
